test/vstest.console.UnitTests/ExecutorUnitTests.cs

```diff
@@ -338,6 +338,25 @@ public void ExecutorShouldPrintRunnerArchitecture()
         Assert.IsFalse(mockOutput.Messages.Any(message => message.Message!.Contains("vstest.console.exe is running in emulated mode")));
     }
 
+    [TestMethod]
+    [DataRow("--ShowDeprecateDotnetVSTestMessage")]
+    [DataRow("--showdeprecateDotnetvTestMessage")]
```

Comment:
Probably:

```suggestion
    [DataRow("--showDeprecateDotNetVSTestMessage")]
```

Reply:
it's done on purpose to test the ordinal ignore casing.

Reply:
never mind me...I'm old.